app/src/androidTest/java/com/droidhats/campuscompass/views/MarkerTest.kt

```diff
@@ -0,0 +1,208 @@
+package com.droidhats.campuscompass.views
+
+import androidx.navigation.testing.TestNavHostController
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import androidx.test.platform.app.InstrumentationRegistry.getInstrumentation
+import androidx.test.rule.ActivityTestRule
+import androidx.test.rule.GrantPermissionRule
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiSelector
+import androidx.test.uiautomator.Direction
+import com.droidhats.campuscompass.MainActivity
+import com.droidhats.campuscompass.R
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/**
+ * Test for buildings initials (Markers) in the map of the MapFragment class.
+ */
+@RunWith(AndroidJUnit4ClassRunner::class)
+class MarkerTest {
+    @get:Rule
+    var activityRule: ActivityTestRule<MainActivity> = ActivityTestRule(MainActivity::class.java)
+    private val twoSeconds: Long = 2000
+    private val fiveSeconds: Long = 5000
```

Comment:
As mentioned earlier for the DirectionTest file. Please apply the same rule here 😄